tabulon_dxf/src/lib.rs

```diff
@@ -428,6 +440,68 @@ pub fn load_file_default_layers(path: impl AsRef<Path>) -> DxfResult<TDDrawing>
         let eh = EntityHandle(NonZeroU64::new(e.common.handle.0).unwrap());
         let lh = handle_for_layer_name[e.common.layer.as_str()];
 
+        let Some(dxf::DrawingItem::Layer(layer)) = drawing.item_by_handle(dxf::Handle(lh.0.get()))
+        else {
+            unreachable!();
```

Comment:
Hmmm.

Reply:
don't you hmm at me Mr. Mitchener :joy:

Reply:
To explain, the `LayerHandle` comes from a `Handle` in a `Layer`, so this always resolves to a `Layer`.